questions/zeldas-quest.md

```diff
@@ -0,0 +1,44 @@
+# Zelda's Quest
+
+## Question
+
+Welcome to **Zelda's Quest**! Here is the problem statement:
+
+You are on a quest to save the kingdom of Hyrule from the evil Ganon. You have
+been tasked with collecting the three pieces of the Triforce, which will give
+you the power to defeat Ganon and restore peace to the land. You have already
+collected two pieces of the Triforce, but you need to find the third piece.
+
+You have received a clue that says: "The third piece of the Triforce is hidden
+in a secret location. To find it, you must solve a puzzle. The puzzle involves a
+list of numbers that are related to the Triforce."
```

Comment:
The last sentence in the clue is hard to read. Why are the numbers related to the triforce? Why does that matter? Could we also merge this sentence into the previous one